src/components/animations/Animation08/WaterRippleBackground.module.css

```
@@ -0,0 +1,416 @@
+  50% {
+    filter: brightness(1.2) hue-rotate(10deg);
+  }
+  100% {
+    filter: brightness(1) hue-rotate(0deg);
+  }
+}
+
+/* Canvas Performance Optimizations */
+.rippleCanvas {
+  image-rendering: auto;
+  image-rendering: crisp-edges;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

image-rendering is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)